__tests__/lib/frontmatter.test.ts

```diff
@@ -0,0 +1,337 @@
+    const data = {
+      title: 'Hello, world!',
+      excerpt: 'This is an excerpt',
+    };
+
+    const result = fix.call(command, data, schema, emptyMappings);
+
+    expect(result.hasIssues).toBe(true);
+    expect(result.updatedData).toMatchInlineSnapshot(`
+      {
+        "content": {
+          "excerpt": "This is an excerpt",
+        },
+        "title": "Hello, world!",
+      }
+    `);
+  });
+
+  it('should fix position', () => {
+    const data = {
+      title: 'Hello, world!',
+      order: 5,
+    };
+
+    const result = fix.call(command, data, schema, emptyMappings);
+
+    expect(result.hasIssues).toBe(true);
+    expect(result.updatedData).toMatchInlineSnapshot(`
+      {
+        "position": 5,
+        "title": "Hello, world!",
+      }
+    `);
+  });
+
+  it('should fix privacy (public)', () => {
+    const data = {
+      title: 'Hello, world!',
+      hidden: false,
+    };
+
+    const result = fix.call(command, data, schema, emptyMappings);
+
+    expect(result.hasIssues).toBe(true);
+    expect(result.updatedData).toMatchInlineSnapshot(`
+      {
+        "privacy": {
+          "view": "public",
+        },
+        "title": "Hello, world!",
+      }
+    `);
+  });
+
+  it('should fix privacy (anyone_with_link)', () => {
+    const data = {
+      title: 'Hello, world!',
+      hidden: true,
+    };
+
+    const result = fix.call(command, data, schema, emptyMappings);
+
+    expect(result.hasIssues).toBe(true);
+    expect(result.updatedData).toMatchInlineSnapshot(`
+      {
+        "privacy": {
+          "view": "anyone_with_link",
+        },
+        "title": "Hello, world!",
+      }
+    `);
+  });
+
+  it.todo('should fix metadata object');
+
+  it.todo('should fix content.link object');
+
+  it.todo('should fix content.next object');
+
+  it('should fix multiple issues', () => {
+    const data = {
+      title: 'Hello, world!',
+      category: '5f92cbf10cf217478ba93561',
+      parentDocSlug: 'some-parent-slug',
+      excerpt: 'This is an excerpt',
+      content: {
+        body: 'This is the body',
+      },
+      order: 7,
+      hidden: true,
+      slug: 'some-slug',
+    };
+
+    const result = fix.call(command, data, schema, {
+      categories: { '5f92cbf10cf217478ba93561': 'some-category-slug' },
+      parentPages: { '5f92cbf10cf217478ba93561': 'some-parent-slug' },
+    });
+
+    expect(result.hasIssues).toBe(true);
+    expect(result.updatedData).toMatchInlineSnapshot(`
+      {
+        "category": {
+          "uri": "some-category-slug",
+        },
+        "content": {
+          "body": "This is the body",
+          "excerpt": "This is an excerpt",
+        },
+        "parent": {
+          "uri": "some-parent-slug",
+        },
+        "position": 7,
+        "privacy": {
+          "view": "anyone_with_link",
+        },
+        "slug": "some-slug",
+        "title": "Hello, world!",
+      }
+    `);
+  });
+});
+
+describe('#fetchSchema', () => {
```

Comment:
May want to move this test case to a different test file because `fetchSchema` is part of `readmeAPIFetch`, not `frontmatter`.

Reply:
ah yeah good catch, ty